wcvendors/wcvendors
# Paypal masspay csv

### All Submissions:


### Changes proposed in this Pull Request:

This adds a PayPal Mass pay button to the commissions screen to download a CSV in the correct format for manually paying vendors via PayPal Mass Pay Web. 

https://developer.paypal.com/docs/payouts/standard/payouts-web/#link-createapaymentfile 

This is the file format we need to output. 

Closes #859 

### How to test the changes in this Pull Request:

1. Set your settings under WC Vendors > Settings > Commissions > PayPal Mass Pay Web 
2. Generate a file on the commissions page 
3. see that format is correct. 

### Other information:
